go.work

```diff
@@ -1,10 +1,12 @@
-go 1.21
+go 1.22
```

Comment:
this strongly affects downstream

Reply:
Yeah it looks like masq requires 1.22 for no real reason?

Reply:
I reverted that but not sure what it will break if a dep requires it? `tidy` must have updated that.